Wireframe/index.html

```diff
@@ -7,27 +7,46 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
+  <header>
```

Comment:
The indentation of this `<header>` seemed much clearer before - why have you re-indented it?